about.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Women In Tech summit</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <script src="https://kit.fontawesome.com/953df531f5.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <!-- NAV -->
+  <main class= "d-flex flex-column mt-5 ">
+    <nav class="bg-dark main-bar">
+      <div class="main-nav d-flex justify-content-end align-items-center">
+        <a href="#"><i class="fa-brands fa-twitter text-light mx-2 mainnav-link"></i></a>
+        <a href="#"><i class="fa-brands fa-facebook text-light mx-2 mainnav-link"></i></a>
+        <a href="#" class="mx-2 nav-link text-light mainnav-link">English</a>
+        <a href="#" class="mx-2 nav-link text-light mainnav-link">My Page</a>
+        <a href="#" class="mx-2 nav-link text-light mainnav-link">Logout</a>
+      </div>
+    </nav>
+  
+  <section class= "bg-white nav-sub">
+    <div class="container-fluid d-flex menu-bar">
+      <a href="./index.html" class="nav-brand px-2 h5">Women tech<br> Summit 2022</a>
+      <ul class=" nav-ul ">
+        <li class="item mx-1 pt-1"><a href="./about.html" class="nav-link h3">About</a></li>
+        <li class="item mx-1 pt-1"><a href="./index.html" class="nav-link h3">Home</a></li>
+        <li class="item mx-1 pt-1"><a href="#" class="nav-link h3">Awards</a></li>
+        <li class="item mx-1 pt-1 "><a href="#" class="nav-link h3">Join</a></li>
+        <li class="item mx-1 pt-1"><a href="#" class="nav-link h3">Sponsor</a></li>
+        <li class="item item-last mx-1 pt-1 "><a href="#" class="nav-link h3">News</a></li>
+
+      </ul> 
+      <div class="hambur my-2">
+        <span class="lin"></span>
+        <span class="lin"></span>
+        <span class="lin"></span>
+      </div> 
+    </div>
+  </section>
+</main>
+<!-- END OF NAV -->
+<!-- Main Section -->
+  <section class="heade">
+    <main class="img-cove ">
+     <!-- <h3 class="h3 text-white-50 container text-center pt-5 pb-1">"Connect with like minds"</h3> -->
```

Comment:
- [Optional] Kindly remove all commented codes from your documents. This will make your work neater.